src/core/print.ml

```diff
@@ -145,7 +155,23 @@ let pp_rule : (sym * pp_hint * rule) pp = fun oc (s,h,r) ->
   let (_, rhs) = Bindlib.unmbind r.rhs in
   Format.fprintf oc "%a → %a" pp lhs pp rhs
 
+(** [pp oc ctx] prints the context [ctx] to the channel [oc]. *)
+let pp_ctxt : ctxt pp = fun oc ctx ->
+  let pp_e oc (x,a,t) =
+    match t with
+    | None    -> Format.fprintf oc "%a:%a" pp_tvar x pp a
+    | Some(t) -> Format.fprintf oc "%a:%a ≔ %a" pp_tvar x pp a pp t
+  in
+  if ctx = [] then Format.pp_print_string oc "∅"
+  else List.pp pp_e ", " oc (List.rev ctx)
+
+(** [wrap_ctxt oc ctx] displays context [ctx] if {!val:print_contexts} is
+    true, with [ ⊢ ] after; and nothing otherwise. *)
+let wrap_ctxt : ctxt pp = fun oc ctx ->
```

Comment:
This name is not well chosen (it doesn't start with pp and wrap is used as a boolean for adding parentheses around). Moreover, pp_ctxt is used nowhere except here. So I propose to merge the code of wrap_ctxt into pp_ctxt, and use pp_ctxt instead of wrap_ctxt everywhere.